app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import shutil
+
+
+class CommandError(Exception):
+    pass
+
+
+def copy_file(command: str) -> None:
+    try:
+        command, input_file, output_file = command.split(" ")
```

Comment:
Check the length of command, it should be exactly 3 arguments, command, input file and output file. And there is no need to pass space to .split(" ").
.split() with no args will work the same because space is default arg here